protocol/v2/ssv/runner/runner_validations.go

```diff
@@ -50,38 +51,59 @@ func (b *BaseRunner) FallBackAndVerifyEachSignature(container *ssv.PartialSigCon
 
 func (b *BaseRunner) ValidatePostConsensusMsg(ctx context.Context, runner Runner, psigMsgs *spectypes.PartialSignatureMessages) error {
 	if !b.hasRunningDuty() {
-		return errors.New("no running duty")
+		return NewRetryableError(spectypes.WrapError(spectypes.NoRunningDutyErrorCode, ErrNoRunningDuty))
 	}
 
-	// TODO https://github.com/ssvlabs/ssv-spec/issues/142 need to fix with this issue solution instead.
-	if len(b.State.DecidedValue) == 0 {
-		return errors.New("no decided value")
+	// slotIsRelevant ensures the post-consensus message is even remotely relevant (eg. we might have already
+	// moved on to another duty that's targeting the next slot but received a post-consensus message relevant
+	// for the duty from the previous slot), this is a relaxed check that helps to filter out inappropriate
+	// messages as soon as possible, the exact slot validation occurs below.
+	slotIsRelevant := func(slot phase0.Slot) error {
```

Comment:
Do we need to do these checks, given that we have message validation that should filter out invalid messages?